src/redux/actions/actions.example.js

```diff
@@ -0,0 +1,7 @@
+
+export const actionExample = (action)=>{
+    return { 
+        type: 'ACTION_TYPE_1',
```

Comment:
You have done well, but I think it might better to have a separate file for types since we will use them in reducers to avoid hard coding them everywhere needed. and another file for actual actions